applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -233,6 +243,26 @@ validate_account_path(Context, AccountId, ?TREE, ?HTTP_GET) ->
         _Else -> Context1
     end.
 
+-spec maybe_get_auth_token(cb_context:context(), ne_binary(), ne_binary()) -> cb_context:context().
```

Comment:
No! The request should be `/v2/accounts/{ACCOUNT_ID}/user_auth/{AUTH_TOKEN}` which means cb_user_auth will get auth_account_id and compare it to `props:get_value(<<"accounts">>, cb_context:req_nouns(Context))`.